### PR TITLE
fix github app manifests docs ✨

### PR DESCRIPTION
**Pull request by hiimbex**
To be blunt, I’m rather mortified that my original screenshot had such an egregious typo. I hope you’ll accept this edit, it uses the updated screenshot from https://github.com/probot/probot/blob/master/docs/assets/probot-setup-wizzard.png and alters the language slightly.

### CODE REVIEW

**Comment by JamesMGreene** [comment]
Always fun to be welcomed by your own bot, right @hiimbex? 😂

**Comment by hiimbex** [comment]
🙈 To be honest, I'm usually more busy being shocked that it's still working! 😂

**Comment by janiceilene** [comment]
Thanks so much @hiimbex! I'll get this triaged for review ⚡

**Comment by janiceilene** [comment]
Thanks for your patience! Our small team is working our way through reviewing all of the amazing contributions ✨

File: content/developers/apps/creating-a-github-app-from-a-manifest.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ To create a Probot App that you can share, follow these steps:
...
 1. Open the project you created, and customize the settings in the `app.yml` file. Probot uses the settings in `app.yml` as the [GitHub App Manifest parameters](#github-app-manifest-parameters).
 1. Add your application's custom code.
-1. [Run the GitHub App locally](https://probot.github.io/docs/development/#running-the-app-locally) or [host it anywhere you'd like](#hosting-your-app-with-glitch). When you navigate to the hosted app's URL, you'll find a web page with a **Register GitHub App** button that people can click to create a preconfigured app. The web page below is Probot's implementation of [step 1](#1-you-redirect-people-to-github-to-create-a-new-github-app) in the GitHub App Manifest flow:
+1. [Run the GitHub App locally](https://probot.github.io/docs/development/#running-the-app-locally) or [host it anywhere you'd like](#hosting-your-app-with-glitch). When you navigate to the hosted app's URL, you'll find a web page with a **Register GitHub App** button that can be clicked to create a preconfigured app. The web page below is Probot's implementation of [step 1](#1-you-redirect-people-to-github-to-create-a-new-github-app) in the GitHub App Manifest flow:
```
Comment (lecoursen) [inline content/developers/apps/creating-a-github-app-from-a-manifest.md:186]: Let's revert this change, as we use active voice whenever possible.

**Review by lecoursen — COMMENTED** [review]
Thanks for updating that image! I left a comment about the text change and will merge the PR as soon as you address it. 😊

**Comment by janiceilene** [comment]
@hiimbex Would you mind reverting the wording change (as suggested by @lecoursen)? Then we can get this merged in 🎉

File: content/developers/apps/creating-a-github-app-from-a-manifest.md (hunk 1 above)
Comment (janiceilene) [reply content/developers/apps/creating-a-github-app-from-a-manifest.md:186]:
```suggestion
1. [Run the GitHub App locally](https://probot.github.io/docs/development/#running-the-app-locally) or [host it anywhere you'd like](#hosting-your-app-with-glitch). When you navigate to the hosted app's URL, you'll find a web page with a **Register GitHub App** button that people can click to create a preconfigured app. The web page below is Probot's implementation of [step 1](#1-you-redirect-people-to-github-to-create-a-new-github-app) in the GitHub App Manifest flow:
```

**Review by janiceilene — APPROVED** [review]
I changed the text back to the original per @lecoursen's suggestion and will merge this once tests are passing 🎉

**Comment by hiimbex** [comment]
Thanks @janiceilene - sorry for not getting back to this one - my github emails these days are too much 🙈
